lib/main.js

```diff
@@ -7,6 +7,7 @@ import config from "./config.json"
 import * as PluginManagement from "./plugin-management"
 import { treeSitterWarning } from "./performance-monitor"
 import DOMStylesReader from "./dom-styles-reader"
+import { debounce } from "underscore-plus"
```

Comment:
Could you please add [the debounce function](https://github.com/jashkenas/underscore/blob/master/modules/debounce.js#L1) to
https://github.com/UziTech/minimap/blob/update-styles/lib/deps/underscore-plus.js

Or directly import it from `underscore`? `underscore-plus` is not tree-shakable